src/pages/home/report/ContextMenu/ContextMenuActions.js

```diff
@@ -120,7 +120,11 @@ export default [
         successTextTranslateKey: '',
         successIcon: null,
         shouldShow: (type, reportAction, isArchivedRoom, betas, anchor, isChronosReport, reportID) =>
-            type === CONTEXT_MENU_TYPES.REPORT_ACTION && reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.ADDCOMMENT && !ReportUtils.isThreadFirstChat(reportAction, reportID),
+            type === CONTEXT_MENU_TYPES.REPORT_ACTION &&
+            !ReportActionUtils.isSplitBillAction(reportAction) &&
+            ((reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.ADDCOMMENT && !ReportUtils.isThreadFirstChat(reportAction, reportID)) ||
+                reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.REPORTPREVIEW ||
+                reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.IOU),
```

Comment:
a bit order change so that isSplitBillAction is not called for every action
```suggestion
            type === CONTEXT_MENU_TYPES.REPORT_ACTION &&
            ((reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.ADDCOMMENT && !ReportUtils.isThreadFirstChat(reportAction, reportID)) ||
                reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.REPORTPREVIEW ||
                (reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.IOU && !ReportActionUtils.isSplitBillAction(reportAction))),
```